src/components/auth-provider.tsx

```diff
@@ -87,21 +87,18 @@ const AuthProvider: React.FC< AuthProviderProps > = ( { children } ) => {
 	const logout = useCallback( async () => {
 		try {
 			if ( ! isOffline && client ) {
-				client.request(
-					{
+				try {
```

Comment:
Do we really need the inner try..catch statement here?

Reply:
Makes sense, done in 31e25f25a961fc531599c332d91a9902a37c87f7